indexers/src/main/java/org/mousephenotype/cda/indexers/StatisticalResultIndexer.java

```diff
@@ -197,6 +249,9 @@ public int processEmbryoResults() throws SQLException, IOException, SolrServerEx
 
 				StatisticalResultDTO doc = parseLineResult(r);
 
+				doc.setDocId(doc.getDocId()+"-"+count);
```

Comment:
Probably unnecessary.  It was to further disambiguate these result IDs from docs of another type, but that turned out not to be the issue with this particular case.  I'll revert it.